$NetBSD: patch-ab,v 1.1 2007/05/17 16:40:19 heinz Exp $

  If use_sha512_hash is not available (with OpenSSL <0.9.8), 4*5 tests will
  not be done.

--- t/rsa.t.orig	2006-11-13 01:01:04.000000000 +0100
+++ t/rsa.t
@@ -4,7 +4,7 @@ use Test;
 use Crypt::OpenSSL::Random;
 use Crypt::OpenSSL::RSA;
 
-BEGIN { plan tests => 45 + (UNIVERSAL::can("Crypt::OpenSSL::RSA", "use_sha512_hash") ? 3*6 : 0) }
+BEGIN { plan tests => 43 + (UNIVERSAL::can("Crypt::OpenSSL::RSA", "use_sha512_hash") ? 4*5 : 0) }
 
 sub _Test_Encrypt_And_Decrypt
 {
